config/routes.rb

```diff
@@ -6,6 +6,8 @@
       resources :cars, only: %i[index create show destroy]
       resources :reservations, only: %i[index create show destroy]
       get '/users/:user_id/reservations', to: 'reservations#user_reservations'
+      post 'users/signin', to: 'sessions#create'
+      get 'users/index'
```

Comment:
👍 SEssion controller Handled

Reply:
@roniy68 Thank you for your review, I have updated all changes. Please have a look!